### PR TITLE
[es5.x] fluentd support for ES 5.x

### PR DESCRIPTION
**Pull request by nhosoi**

### CODE REVIEW

**Comment by nhosoi** [comment]
[(5) update fluentd to work with ES5.x [logging][epic-ois-5]](https://trello.com/c/tOPrdf4n/548-5-update-fluentd-to-work-with-es5x-loggingepic-ois-5)

**Comment by nhosoi** [comment]
Tests are failing with these fatal errors.
```
[FATAL] Elasticsearch pod is missing expected plugin(s). Exp cloud-kubernetes, openshift-elasticsearch, prometheus-exporter, found: discovery-kubernetes
[FATAL] openshift-elasticsearch
[FATAL] prometheus-exporter
[ERROR] PID 77499: hack/testing/check-logs.sh:33: `USE_JOURNAL="${USE_JOURNAL}" OAL_ELASTICSEARCH_COMPONENT="es" OAL_KIBANA_COMPONENT="kibana" OAL_ELASTICSEARCH_SERVICE="logging-es" "${OS_O_A_L_DIR}/test/cluster/functionality.sh"` exited with status 1.
```

**Comment by nhosoi** [comment]
@richm Could you please review this pr?  
Although the Logging tests are failing, it seems I'm allowed to "Merge pull request"...?  (The Merge pull request button is not greyed out...)

File: fluentd/Dockerfile.centos7 [hunk 1]
```diff
@@ -35,11 +35,11 @@ RUN mkdir -p ${HOME} && mkdir -p /etc/fluent/plugin && \
...
      'public_suffix:<3' \
-     'elasticsearch-transport:<5' \
-     'elasticsearch-api:<5' \
-     'elasticsearch:<5' \
+     'elasticsearch-transport:<6' \
+     'elasticsearch-api:<6' \
+     'elasticsearch:<6' \
       fluentd:${FLUENTD_VERSION} \
-     'fluent-plugin-elasticsearch:~>1.0' \
+     'fluent-plugin-elasticsearch:<2.0.0' \
```
Comment (richm) [inline fluentd/Dockerfile.centos7]: shouldn't this be `fluent-plugin-elasticsearch:~>2.0` ?

Comment (nhosoi) [reply fluentd/Dockerfile.centos7]: According to the readme of fluent-plugin-elasticsearch [1], fluent-plugin-elasticsearch has to be older than 2.0.0 for fluentd v0.12?
```
fluent-plugin-elasticsearch | fluentd      | ruby
        >= 2.0.0            | >= v0.14.20  | >= 2.1
        < 2.0.0             | >= v0.12.0   | >= 1.9
```
[1] https://github.com/uken/fluent-plugin-elasticsearch/

Comment (richm) [reply fluentd/Dockerfile.centos7]: ok

**Comment by nhosoi** [comment]
Thanks, @richm !

**Pull request merged**